.github/CONTRIBUTING.md

```diff
@@ -2,7 +2,7 @@
 Bugcrowd welcomes community feedback and direct contributions to the Bugcrowd VRT. We accept comments for public discussion via GitHub Issues, but can also accommodate comments made via email to [vrt@bugcrowd.com](mailto:vrt@bugcrowd.com).
 
 ## Process
-Please open your feedback as an **Issue** and label it as either a `bug` or an `enhancement`. The Bugcrowd team strives to review and comment on new Issues within three business days. Large or systemic changes should first be discussed in an Issue rather than be submitted as a pull request directly. If you have a suggested minor change that includes:
+Please open your feedback as an **Issue**. The Bugcrowd team strives to review and comment on new Issues within three business days. Large or systemic changes should first be discussed in an Issue rather than be submitted as a pull request directly. If you have a suggested minor change that includes:
 
```

Comment:
Can we also change 3 business days to 5 business days to give the team a chance to discuss issues during the vulnerability roundtable?